src/udp/socket.rs

```diff
@@ -1007,7 +1007,6 @@ mod test {
                 .into_future()
                 .map_err(|(e, _)| panic!("recv error: {}", e))
                 .and_then({
-                    let sock_pk = sock_pk.clone();
```

Comment:
yeah, `PublicKey` implements `Copy` trait :)

Reply:
Weird that it does but `SecretKey` doesn't. Though I guess it's slightly more secure not being able to copy a private key all over your program.

Reply:
Not sure what that buys us from security perspective because we clone it all the time anyway :D